flake.nix

```
@@ -0,0 +1,21 @@
+{
+  description = "vs-overlay";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  outputs = { self, nixpkgs }:
+    let
+      systems = [ "x86_64-linux" "aarch64-linux" "i686-linux" "x86_64-darwin" ];
+      forAllSystems = f: nixpkgs.lib.genAttrs systems (system: f system);
+      nixpkgsFor = forAllSystems (system:
+        import nixpkgs {
+          inherit system;
+          overlays = [ self.overlay ];
+        }
+      );
+    in
+    {
+      overlay = import ./default.nix;
+      packages = forAllSystems (system: {
+        inherit (nixpkgsFor.${system}) vapoursynthPlugins;
```

```suggestion
        inherit (nixpkgsFor.${system})
          getnative
          vapoursynthPlugins;
```
`getnative` is currently not accessible via the `packages` output